.gitignore

```diff
@@ -1,3 +1,12 @@
 /node_modules
 /test/multidep
 /tmp
+
+**/*.js
+**/*.d.ts
```

Comment:
sounds like you intended to have some exceptions, and check in some  `.d.ts` files